### PR TITLE
Implement GAX Metrics API helper.

### PR DESCRIPTION
**Pull request by tseaver**
~~Uses #1887 as a base.~~

### CODE REVIEW

File: gcloud/logging/_gax.py [hunk 1]
```diff
@@ -113,6 +121,274 @@ def logger_delete(self, project, logger_name):
         self._gax_api.delete_log(path, options)
 
 
+class _SinksAPI(object):
```
Comment (daspecster) [inline gcloud/logging/_gax.py]: Isn't this from #1887?

Comment (tseaver) [inline gcloud/logging/_gax.py]: Yes, this one is based on #1887.  I will rebase it after merging.

Comment (tseaver) [inline gcloud/logging/_gax.py]: Now rebased.

**Comment by tseaver** [comment]
@daspecster Any remaining issues?

**Comment by daspecster** [comment]
LGTM!

**Pull request merged**